<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>129</y>
      <width>300</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="zaslon_dnar">
     <property name="text">
      <string>Denar:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="piskotek">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>463</width>
      <height>541</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>225</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>459</width>
          <height>367</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="kurzorjipp">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Kurzor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_kurzorja">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena kurzorja:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="babicapp">
               <property name="text">
                <string>Babica</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_babice">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena babice:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="farmapp">
               <property name="text">
                <string>Farma</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_farme">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena farme:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="tovarnapp">
               <property name="text">
                <string>Tovarna</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_tovarne">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena tovarne:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="rudnikpp">
               <property name="text">
                <string>Rudnik</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_rudnika">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena rudnika:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="posiljkapp">
               <property name="text">
                <string>Pošiljka</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_posiljke">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena pošiljke:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="laboratorijpp">
               <property name="text">
                <string>Laboratorij</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_laboratorija">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena laboratorija:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QPushButton" name="portalpp">
               <property name="text">
                <string>Portal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_portala">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena portala:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="cas_strojpp">
               <property name="text">
                <string>Časovni stroj</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_cas_stroja">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena časovnega stroja:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QPushButton" name="antipp">
               <property name="text">
                <string>Antimaterijski
kondenzator</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_anti">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena antimaterijskega kondenzatorja:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="prizmapp">
               <property name="text">
                <string>Prizma</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cena_prizme">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cena prizme:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>225</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>459</width>
          <height>223</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Ojačan kazalec (100 piškotkov)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Nadgradi_piskotek_klik">
             <property name="text">
              <string>Nadgradi</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Krema za preprečevanje karpalnega kanala (400 piškotkov)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Nadgradi_kremo_kanal">
             <property name="text">
              <string>Nadgradi</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>piskotek</zorder>
   <zorder>zaslon_dnar</zorder>
   <zorder>scrollArea</zorder>
   <zorder>scrollArea_2</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
